examples/vault-auto-unseal/README.md

```diff
@@ -56,6 +55,49 @@ we strongly recommend deploying the Vault cluster into the private subnets of a
   `vault write /sys/license "text=$LICENSE_KEY_TEXT"`. If you don't do that, Vault
   will re-seal after 30 minutes and you won't be able to write or read secrets anymore.
 
+### Seal
+
+Vault servers boot sealed, which means it can access its storage, but can't decrypt it.
+So you can't really do anything apart from unsealing it or checking the server status.
+When you unseal, it recreates the master key, which isn't stored anywhere and is
+used to read the stored decryption key, which then can decrypt the data, and then
+you can start performing other operations on Vault.
```

Comment:
You might make this explanation more clear by first stating that all data stored by Vault is encrypted with a Master Key which Vault only ever keeps in memory (technically, there are "intermediate keys" used to encrypt the data and it's the intermediate keys that are decrypted with the Master Key, but that detail doesn't matter here). When Vault first boots, it does not have the Master Key in memory, and therefore cannot decrypt its own data, limiting its functionality to checking the server status or writing the Master Key to memory. In such a state, Vault is "sealed."